add documentation about using `event` in `getVar`

yuhui opened the pull request:
This is already documented at https://github.com/AdobeDocs/experience-platform.en/edit/main/help/tags/ui/managing-resources/data-elements.md, but also adding it here for users who might not refer to that other page.

djwendo commented:
Thank you for submitting this contribution. We will review and respond asap.

Joshua-McGee commented:
@arobsco Thank you for submitting the report. This has been resolved and the documentation has been updated: https://docs.adobe.com/content/help/en/experience-platform/edge/fundamentals/debugging.html#retrieving-library-information

yuhui committed:
add documentation about using `event` in `getVar`

This is already documented at https://github.com/AdobeDocs/experience-platform.en/edit/main/help/tags/ui/managing-resources/data-elements.md, but also adding it here for users who might not refer to that other page.
